app/create_app.py

```diff
@@ -114,22 +114,25 @@ def inject_global_constants():
         )
 
     @flask_app.context_processor
-    def inject_service_name():
-        fund = None
-        if request.view_args or request.args or request.form:
-            try:
-                fund = find_fund_in_request()
-            except Exception as e:  # noqa
-                current_app.logger.warning(
-                    f"""Exception: {e}, occured when trying to
-                    reach url: {request.url}, with view_args:
-                    {request.view_args}, and args: {request.args}"""
-                )
-        if fund:
-            service_title = gettext("Apply for") + " " + fund.title
-        else:
-            service_title = gettext("Access Funding")
-        return dict(service_title=service_title)
+    def utility_processor():
+        def _get_service_title():
+            fund = None
+            if request.view_args or request.args or request.form:
+                try:
+                    fund = find_fund_in_request()
+                except Exception as e:  # noqa
+                    current_app.logger.warning(
+                        f"""Exception: {e}, occured when trying to
+                        reach url: {request.url}, with view_args:
+                        {request.view_args}, and args: {request.args}"""
```

Comment:
The triple-quote string here is weird (but I recognise that's a carryover 👀)

Reply:
Changed it to use %s